test/nootils.spec.js

```diff
@@ -99,6 +99,17 @@ describe('Utils', () => {
       expect(actual._id).to.eq(3);
     });
 
+    it('returns most recent matching report from multiple forms', () => {
+      const reports = [
+        { _id: 1, form: 'H', reported_date: 1 },
+        { _id: 2, form: 'V', reported_date: 2 },
+        { _id: 3, form: 'V', reported_date: 3 },
+        { _id: 4, form: 'H', reported_date: 4 }
```

Comment:
To improve functional coverage, please add some reports that don't match the forms, so we make sure those are not returned as results. Also, please shuffle `reports` so we don't pass an ordered list. 
Something like: 
```
    const reports = [
        { _id: 6, form: 'B', reported_date: 9 },
        { _id: 3, form: 'V', reported_date: 3 },
        { _id: 4, form: 'H', reported_date: 4 },
        { _id: 2, form: 'V', reported_date: 2 },
        { _id: 5, form: 'A', reported_date: 6 },
        { _id: 1, form: 'H', reported_date: 1 },
        { _id: 7, form: 'C', reported_date: 12 },
      ];
```